wikichange/src/content/tagEveryWord.js

```diff
@@ -12,9 +12,9 @@ const tagEveryWord = () => {
                 return NodeFilter.FILTER_REJECT;
             }
             // Skip the references
-            if (node.parentElement.closest('.reflist')){
-                return NodeFilter.FILTER_REJECT;
-            }
+            // if (node.parentElement.closest('.reflist')){
+            //     return NodeFilter.FILTER_REJECT;
+            // }
```

Comment:
do we still need this?

Reply:
Yes, @etiennerichart told me we should keep this in case of heavier pages like US